test/ConsolePlayerTest.java

```diff
@@ -0,0 +1,11 @@
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConsolePlayerTest {
+    @Test
+    public void createComputerPlayerWithAThrow() {
+        ConsolePlayer consolePlayer = new ConsolePlayer(Throw.ROCK);
+        assertEquals(Throw.ROCK, consolePlayer.getThrow());
```

Comment:
Interestingly that is where I got to last night!  See my last commit/PR on the game branch I think..